src/main/java/com/commercetools/emailprocessor/model/Statistics.java

```diff
@@ -0,0 +1,125 @@
+package com.commercetools.emailprocessor.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+
+
+public class Statistics {
+    public static final int RESPONSE_CODE_SUCCESS = 200;
+    public static final int RESPONSE_ERROR_TEMP = 503;
+    public static final int RESPONSE_ERROR_PERMANENT = 400;
+    public static final int RESPONSE_IGNORED = 0;
+    private String tenantId = "";
+    private int globalError = 0;
+    private int processed = 0;
+    private int notProcessed = 0;
+    private int sentSuccessfully = 0;
+    private int permanentErrors = 0;
+    private int temporarilyErrors = 0;
```

Comment:
Please change the variable name to `temporaryErrors` @ahalberkamp

Reply:
803f03f